webapp/schemas.py

```diff
@@ -173,5 +191,26 @@ class CVEImportSchema(CVESchema):
 class CVEAPISchema(CVESchema):
     package_statuses = List(Nested(CvePackage), data_key="packages")
     notices_ids = List(
-        String(validate=Regexp(r"USN-\d{1,5}-\d{1,2}")), required=False
+        String(validate=Regexp(r"(USN|LSN)-\d{1,5}-\d{1,2}")),
+        data_key="notices",
     )
+
+
+class CVEsAPISchema(Schema):
+    cves = List(Nested(CVEAPISchema))
+    offset = Int(allow_none=True)
+    limit = Int(allow_none=True)
+    total_results = Int()
+
+
+# TODO: This should be a Schema object, but parameters won't load that way
```

Comment:
I am not sure what it means exactly.
If you put a TODO could you add and issue that links to this code and explains what you would like to do with this code?

Reply:
Created the issue and linked it under the comment.